test/system/users_test.rb

```diff
@@ -0,0 +1,119 @@
+require "application_system_test_case"
+
+class UsersTest < ApplicationSystemTestCase
+  test "Invite, register, confirm, promote, demote, delete" do
+    admin = users(:admin_user)
+    viewer_email = 'user@example.com'
+
+    #
+    # Administrator sends a user invitation
+    #
+    Capybara.using_session('admin') do
```

Comment:
Not an issue; just curious: why the switch from symbols to strings?

Reply:
That was a holdover from debugging the session logout problem. I changed it back to symbols.